features/google/src/locales/en/explore.json

```diff
@@ -3,5 +3,5 @@
     "reportCard.text": "If you send us an anonymised report about the structure of your Google data, it would help us improve the Google Data Importer so that it can show you even more insights.",
     "reportCard.button": "Learn more",
     "report.success": "Your report was sent to us. Thank you!",
-    "report.error": "Your report could not be sent. Please try again!"
+    "report.error": "Report could not be sent. Try again!"
```

Comment:
We should definitely have common strings to not repeat the same messages over and over again...